blaze/compute/pandas.py

```diff
@@ -523,11 +533,14 @@ def compute_up(t, df, scope=None, **kwargs):
     return pd.concat(children, axis=1)
 
 
-@dispatch(Summary, DataFrame)
+@dispatch(Summary, (DataFrame, DaskDataFrame))
 def compute_up(expr, data, **kwargs):
     values = [compute(val, {expr._child: data}) for val in expr.values]
     if expr.keepdims:
-        return DataFrame([values], columns=expr.fields)
+        if isinstance(data, Dataframe):
```

Comment:
this is also a typo `Dataframe` should be `DataFrame` ... i think `type(self)(...)` would solve the problem